pages/dkp/kommander/2.1/install/configuration/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.pug
+navigationTitle: Configuration
+title: Configuration
+menuWeight: 30
+excerpt: How to configure a Kommander installation
+beta: false
+enterprise: false
+---
+
+Kommander can be configured during the initial installation and also post-installation using the Kommander CLI.
```

Comment:
```suggestion
You can configure Kommander during the initial installation, and also post-installation using the Kommander CLI.
```

Reply:
resolved in a2389daa9fd58b84770bb55c6ee73e7b4e829e54